autocertkit/ack_cli.py

```diff
@@ -188,34 +184,49 @@ def parse_netconf_file(filename):
         key, value = [items.strip() for items in arr.split('=')]
 
         if key.startswith('eth'):
-            # Each line is in the format:
-            # ethX = 3,[235,236,237]
-            csv = [v.strip() for v in value.split(',')]
-
             # Ethernet Interface
             utils.log.debug("Ethernet Interface: '%s'" % key)
 
+            # Each line is in the format:
+            # ethX = 3,[235,236,237],igbvf,igbvf-2.3.9.6-1.x86_64.rpm # comment
```

Comment:
At my first glance to the network conf file, I don't like it either. I agree with you 100%, while my only concern is that the file contains not only Ethernet info ('eth') but also static ip info ('static' and "static_management"), I need effort to do this changes, also re-write UT.
Therefore, I think I am able to refactor later (after current PR merged), because the deadline is coming.

Reply:
OK - but can you confirm that these arguments are not _required_? I.e. it's optional to have the VF information in your network.conf file?

Reply:
Yes, but the delimiter, comma, should keep there.
E.g. If VF driver is already existed in LDVM (vendor may use prepared DLVM) then config like:
   ethX = 3,[235,236,237],,